Sources/TuistKit/Services/ImplicitImportsLintService.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Path
+import TuistCore
+import TuistLoader
+
+class ImplicitImportsLintService {
```

Comment:
Let's add some basic tests for this one

Reply:
What kind of tests do you want to add? LintImplisitCommand is tested and all services inside of it are tested. I can do similar to inner test or something else?

Reply:
We shouldn't be testing `LintImplicitCommand` at all in unit tests as all the business logic should be in the `LintImplicitService`.

Reply:
We tested all Services inside of LintImplicitService as well, we made Acceptance tests on LintImplicitCommand. I dont get what kind if tests can be done on LintImplicitService